_posts/0000-01-02-ervinkurbegovic.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
we love rock'n'roll and we gone never stop ist **we love rock'n'roll** _yeahhhh_
```suggestion
maybe jazz is cool too
```